doc/user_guide/marks/geoshape.rst

```diff
@@ -0,0 +1,689 @@
+
+You can use the ``lookup`` transform in two directions:
+
+1. Use a ``GeoDataFrame`` with geometries as source and lookup related information in another ``DataFrame``.
+2. Use a ``DataFrame`` as source and lookup related geometries in a ``GeoDataFrame``.
+
+Depending on your use-case one or the other is more favorable.
+
+First we show an example of the first approach. 
+Here we lookup the field ``rate`` from the ``df_us_unemp`` DataFrame, where the ``gdf_us_counties`` GeoDataFrame is used as source: 
+
+.. altair-plot::
+
+    import altair as alt
+    from vega_datasets import data
+    import geopandas as gpd
+
+    gdf_us_counties = gpd.read_file(data.us_10m.url, driver='TopoJSON', layer='counties')
+    df_us_unemp = data.unemployment()
+
+    alt.Chart(gdf_us_counties).mark_geoshape().transform_lookup(
+        lookup='id', 
+        from_=alt.LookupData(data=df_us_unemp, key='id', fields=['rate'])
+    ).encode(
+        alt.Color('rate:Q')
+    ).project(
+        type='albersUsa'
+    )
+
+Next, we show an example of the second approach.
+Here we lookup the geometries through the fields ``geometry`` and ``type`` from the ``gdf_us_counties`` GeoDataFrame, where the ``df_us_unemp`` DataFrame is used as source.
+
+.. altair-plot::
+
+    alt.Chart(df_us_unemp).mark_geoshape().transform_lookup(
+        lookup='id',
+        from_=alt.LookupData(data=gdf_us_counties, key='id', fields=['geometry', 'type'])
+    ).encode(
+        alt.Color('rate:Q')
+    ).project(
+        type='albersUsa'
+    )
+
+
+Chloropleth Classification
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+Chloropleth maps provide an easy way to visualize how a variable varies across a 
+geographic area or show the level of variability within a region. 
+
+We first define a utility function ``classify()`` that we will use to showcase different approaches to make a chloropleth map.
+We apply it to define a chloropleth map of the unemployment statistics of 2018 of US counties using a ``linear`` scale. 
+
+.. altair-plot::
+
+    import altair as alt
+    from vega_datasets import data
+    import geopandas as gpd
+
+    def classify(scale_type, breaks=None, nice=False, title=None, size='small'):
+
+        if size =='default':
+            width=400
+            height=300
+        else:
+            width=180
+            height=130
+            
+        # us_counties = gpd.read_file(data.us_10m.url, driver='TopoJSON', layer='counties')
+        # us_states = gpd.read_file(data.us_10m.url, driver='TopoJSON', layer='states')
+        us_counties = alt.topo_feature(data.us_10m.url, 'counties')
+        us_states = alt.topo_feature(data.us_10m.url, 'states')
+        us_unemp = data.unemployment.url
+        
+        if title is None:
+            title=scale_type
+        
+        if 'threshold' in scale_type:
+            scale = alt.Scale(type=scale_type, domain=breaks, scheme='turbo')
+        else:
+            scale = alt.Scale(type=scale_type, nice=nice, scheme='turbo')
```

Comment:
```suggestion
            scale = alt.Scale(type=scale_type, domain=breaks, scheme='inferno')
        else:
            scale = alt.Scale(type=scale_type, nice=nice, scheme='inferno')
```

What do you think about changing the color scale to another multi-hue sequential map that has a more intuitive color change from low to high values? I know turbo is better than jet and the like but I think it is still problematic since it doesn't increase uniformly in lightness, whereas e.g. inferno does and I think it looks pretty good:

![image](https://user-images.githubusercontent.com/4560057/200103697-8dcc7852-a5cf-404e-a9a6-6b493449f1ba.png)

Compared with the current turbo:

![image](https://user-images.githubusercontent.com/4560057/200104118-1fb8ca41-4a2b-493d-8424-80d319d431ec.png)


`viridis` looks more boring since it doesn't quite reach into the yellow values at the high end:

![image](https://user-images.githubusercontent.com/4560057/200104003-1868a89b-09db-418f-9775-881c852c269c.png)